src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/Serialization/SymbolSpecification.cs

```diff
@@ -348,6 +369,43 @@ internal XElement CreateXElement()
                    MethodKind.HasValue ? new XElement(nameof(MethodKind), MethodKind) :
                    throw ExceptionUtilities.Unreachable;
 
+            public bool ShouldReuseInSerialization => throw new NotImplementedException();
```

Comment:
Should this be returning `false` like other implementations?

Reply:
indeed.